dspace-api/src/main/java/org/dspace/content/crosswalk/METSRightsCrosswalk.java

```diff
@@ -471,85 +471,31 @@ else if (element.getName().equals("Context"))
                         {
-                            rp.setRpName(element.getAttributeValue("rpName"));
-                        }
-                        try {
-                            if (element.getAttributeValue("start-date") != null)
-                            {
-                                rp.setStartDate(sdf.parse(element.getAttributeValue("start-date")));
-                            }
-                            if (element.getAttributeValue("end-date") != null)
-                            {
-                                rp.setEndDate(sdf.parse(element.getAttributeValue("end-date")));
-                            }
-                        }catch (ParseException ex) {
-                            java.util.logging.Logger.getLogger(METSRightsCrosswalk.class.getName()).log(Level.SEVERE, null, ex);
+                        	rp.setEndDate(sdf.parse(element.getAttributeValue("end-date")));
                         }
-
-                        List<Element> le = new ArrayList<Element>(element.getChildren());
-                        for (Element el : le)
-                        {
-                            if ((el.getAttributeValue("DISCOVER").equalsIgnoreCase("true")) 
-                                    && (el.getAttributeValue("DISPLAY").equalsIgnoreCase("true")))
-                            {
-                                if (el.getAttributeValue("DELETE").equalsIgnoreCase("false"))
-                                {
-                                    if (el.getAttributeValue("MODIFY").equalsIgnoreCase("false"))
-                                    {
-                                        rp.setAction(Constants.READ);
-                                    }
-                                    else
-                                    {
-                                        rp.setAction(Constants.WRITE);
-                                    }
-                                }
-                                else
-                                {
-                                    if (el.getAttributeValue("MODIFY").equalsIgnoreCase("true"))
-                                    {
-                                        rp.setAction(Constants.DELETE);
-                                        if ((el.getAttributeValue("COPY").equalsIgnoreCase("true"))
-                                                &&(el.getAttributeValue("DUPLICATE").equalsIgnoreCase("true"))
-                                                &&(el.getAttributeValue("PRINT").equalsIgnoreCase("true")))
-                                        {
-                                            rp.setAction(Constants.ADMIN);
-                                        }
-                                    }
-                                }
-                            }
-                        }
-                        policies.add(rp);
-                    } catch (NullPointerException ex) {
```

Comment:
Unclear to me why this entire `for` loop has been stripped out?  Is some other area of the class achieving this same logic?